pom.xml

```diff
@@ -20,7 +20,7 @@
 		<java.version>1.8</java.version>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<spring.framework.version>4.3.28.RELEASE</spring.framework.version>
-		<license.maven.plugin.version>4.0.rc2</license.maven.plugin.version>
+		<license.maven.plugin.version>4.0</license.maven.plugin.version>
```

Comment:
I applied this change and ran the license update and 174 files were correctly updated to include the end year of "2021" that were updated during "2021".   LGTM.  Approved for merge.

Reply:
Thanks!